tests/integration/targets/rds_cluster_modify/tasks/main.yaml

```diff
@@ -177,7 +183,7 @@
 
   - name: Create DB cluster parameter group if not exists
     command: aws rds create-db-cluster-parameter-group --db-cluster-parameter-group-name
-      {{ new_db_parameter_group_name }} --db-parameter-group-family aurora-mysql5.7 --description
+      {{ new_db_parameter_group_name }} --db-parameter-group-family aurora-mysql8.0 --description
```

Comment:
note: updated the engine version due to following failure seen in integration tests

```

2023-09-11 20:41:13.346386 | controller |   "msg": "Unable to modify DB cluster: An error occurred 
(InvalidParameterCombination) when calling the ModifyDBCluster operation: The Parameter Group 
ansible-test-db-parameter-group-e096d2ecba72-new with DBParameterGroupFamily aurora-mysql5.7 
cannot be used for this instance. Please use a Parameter Group with DBParameterGroupFamily
 aurora-mysql8.0",


```